engines/pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtNDArrayEx.java

```diff
@@ -694,7 +696,59 @@ public NDList multiBoxPrior(
             List<Float> steps,
             List<Float> offsets,
             boolean clip) {
```

Comment:
clip is not been used in your implementaiton

Reply:
Thanks - agree it is not used. I'm asking for help here - and reading the baseline implementation in mxnet. 

Looking at the baseline code:
https://github.com/apache/mxnet/blob/master/src/operator/contrib/multibox_prior.cc#L53
We don't see `clip` used there either. (Correct me if I'm wrong)

We do see a reference to `clip` in the interface for this java method so we have to implement it as part of our class. 
https://github.com/deepjavalibrary/djl/blob/master/api/src/main/java/ai/djl/ndarray/internal/NDArrayEx.java#L601
 
There are hints of an implementation using `clip` in old mxnet code:
https://github.com/apache/mxnet/blob/a720b15b5fa011a9610dfaeabf0792443c6abec5/src/operator/contrib/multibox_prior-inl.h#L128
But it appears to have been removed.